### PR TITLE
Develop swagger

### PR DESCRIPTION
**Pull request by virichino**
-Add swagger to document the apis from video converter and machine learning

### CODE REVIEW

File: RESEARCH/SWAGGER/db.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      "title": "The New Turing Omnibus",
+      "author": "Alexander K. Dewdney"
+    },
+    {
+      "id": "d5fE_asz",
+      "title": "The New Turing Omnibus",
+      "author": "Alexander K. Dewdney"
+    }
+  ]
+}
```
Comment (serqiiiio) [inline RESEARCH/SWAGGER/db.json:24]: last line needed

File: SWAGGER/.env [hunk 2]
```diff
@@ -0,0 +1,4 @@
+HOSTNAME = localhost
+SERVICE_PORT = 8000
+CONVERTER_PORT = 8080
+ML_PORT = 8085
```
Comment (serqiiiio) [inline SWAGGER/.env:4]: last line required

**Review by serqiiiio — CHANGES_REQUESTED** [review]

File: RESEARCH/SWAGGER/db.json (hunk 1 above)
Comment (virichino) [reply RESEARCH/SWAGGER/db.json]: Done

File: SWAGGER/.env (hunk 2 above)
Comment (virichino) [reply SWAGGER/.env]: Done

**Review by serqiiiio — APPROVED** [review]

**Pull request merged**